packages/cli/src/commands/whoami.ts

```diff
@@ -0,0 +1,68 @@
+import { z } from "zod";
+import { telemetryClient } from "../telemetry/telemetry.js";
+import { logger } from "../utils/logger.js";
+import { resolvePath } from "../utils/parseNameAndPath.js";
+import { TriggerApi } from "../utils/triggerApi.js";
+import { DevCommandOptions, getEndpointIdFromPackageJson, getTriggerApiDetails } from "./dev.js";
+import ora from "ora";
+
+export const WhoAmICommandOptionsSchema = z.object({
+  envFile: z.string()
+});
+
+export type WhoAmICommandOptions = z.infer<typeof WhoAmICommandOptionsSchema>;
+
+export async function whoamiCommand(path: string, anyOptions: any) {
+  const loadingSpinner = ora(`Hold while we fetch your data`);
+  loadingSpinner.start();
+
+  telemetryClient.dev.started(path, anyOptions);
```

Comment:
Remove all the telemetry calls from this file. We use them to track that the commands are running but only for critical commands, and this one is less important to track.

Reply:
@matt-aitken Done, just pushed the commit.